api/services/organization/Organization.js

```diff
@@ -20,6 +20,10 @@ function throwError(message) {
   throw new CustomError(message, 422);
 }
 
+function hasManager(org, user) {
+  return org.OrganizationRoles.some(or => or.User.id === user.id && or.Role.name === 'manager');
```

Comment:
an `OrganizationRole` must have a `Role` and a `User` since it's just the join table, if this fails as is something has gone horribly wrong so i don't think we should handle that case.